src/07-even-more-loops.js

```diff
@@ -12,7 +12,14 @@
  *  shoutForLoop([ "A", "Very", "Happy", "Array" ]);
  *  //> [ "A!", "Very!", "Happy!", "Array!" ];
  */
-function shoutForLoop(array) {}
+function shoutForLoop(array) {
+  let newArr = []
+  for(let i = 0; i <= array.length-1; i++){
+    shout = array[i] + "!"
```

Comment:
be verry careful on 18 - ` shout ` is now a global variable as there is no ` let ` or ` const `.